cobigen-templates/templates-devon4j-tests/src/test/resources/utils/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.devonfw.cobigen.templates</groupId>
+  <artifactId>templates-devon4j-utils</artifactId>
+  <version>2021.12.007-SNAPSHOT</version>
+  <name>CobiGen Devon4j Templates Utils</name>
+  <description>A Code-based incremental Generator</description>
+  <url>https://github.com/devonfw/cobigen/templates-parent/templates-devon4j-utils</url>
```

Comment:
Link is not working

Reply:
this file should be removed in each template set